.github/workflows/parse_changed_files.py

```diff
@@ -0,0 +1,86 @@
+import re
+import sys
+from pathlib import Path
+from typing import List, Tuple
+
+PLUGIN_DIRS = ['benchmarks', 'data', 'models', 'metrics']
```

Comment:
```suggestion
PLUGIN_DIRS = ['models', 'benchmarks', 'data', 'metrics']
```
keeping the last three together